### PR TITLE
Add date information to time table

### PR DESCRIPTION
**Pull request by timzimmermann**
Add information about the date as well as unix_time and an ISO 8601-formatted string to the table.

See #1297.

### CODE REVIEW

**Comment by osqueryer** [comment]
Can one of the osquery admins reply with "ok to test" to kick off a build...

File: osquery/tables/utility/time.cpp [hunk 1]
```diff
@@ -15,13 +15,25 @@
 namespace osquery {
 namespace tables {
 
+const char* WEEKDAYS[] = {"Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"};
```
Comment (theopolis) [inline osquery/tables/utility/time.cpp]: Mind switching this to a string vector, and using something like `kTimeWeekdays`? https://github.com/facebook/osquery/blob/master/osquery/tables/system/darwin/launchd.cpp#L27

Though you might be able to grab the weekday string from C++'s `asctime`'s formats... maybe? I wouldn't search to long, a const vector is cool for now.

File: osquery/tables/utility/time.cpp [hunk 2]
```diff
@@ -15,13 +15,25 @@
...
+
 QueryData genTime(QueryContext& context) {
   Row r;
   time_t _time = time(0);
   struct tm* now = localtime(&_time);
+  char timestamp[21];
+  strftime(timestamp, sizeof(timestamp), "%FT%TZ", now);
```
Comment (theopolis) [inline osquery/tables/utility/time.cpp]: Will this be zulu time? I believe you'll need to convert local time to gmtime first.

Another thing I just remembered: in the schedule log there's a `calendarTime` field with the following format "Sat Jul  4 01:53:53 2015 UTC" and implementation: https://github.com/facebook/osquery/blob/master/osquery/core/system.cpp#L84

Maybe we should be consistent and call `timestamp` this calendar representation. The ISO representation is IMO "the best" but our existing format is (A) existing :/, and (B) the default from C++'s `asctime`.

**Comment by timzimmermann** [comment]
Check 77604b0.
I added both the ISO 8601 as well as the `asctime()` format. That way, a possible switch to the ISO 8601 format should be facilitated in the future.
I also finally found the way to get the weekday directly from a function in C++.

**Comment by theopolis** [comment]
Looks great! If you don't mind adding those two initializers for the char arrays we'll merge with fire!

**Comment by timzimmermann** [comment]
Awesome, I guess it is ready now.

**Comment by theopolis** [comment]
ok to test

**Pull request merged**